aoc-2022-rust/src/days/day_13/main_13.rs

```diff
@@ -61,9 +72,13 @@ fn day_12_part_1() {
     let input = include_str!("input");
     let result = part_1(input);
     println!("{}", result);
+    // I would put a more specific assert message (I would include the amount of "trues" in the
+    // message below for example)
     assert!(result < 5545, "too many trues, more cases should fail");
     assert!(result > 3000, "guess attempt, should be higher then 3000");
     assert!(result > 4500, "guess attempt, should be higher then 3000");
+
+    // The first 3 asserts seem unnecessary with this one
```

Comment:
yeah indeed, these were test cases I added because I entered a wrong result hihi
rermoved it